DrivewayDentDeletion/Operators/prereqs.sh

```diff
@@ -105,37 +135,57 @@ else
-kind: Configuration
-metadata:
-  name: ace-policyproject
-  namespace: ${namespace}
-spec:
-  contents: "$temp"
-  type: policyproject
-"
-echo "${configyaml}" > ${PWD}/tmp/policy-project-config.yaml
-echo "INFO: Output -> policy-project-config.yaml"
-cat ${PWD}/tmp/policy-project-config.yaml
-oc apply -f ${PWD}/tmp/policy-project-config.yaml
+# setting up policyporject for both namespaces
+declare -a image_projects=("${dev_namespace}" "${test_namespace}")
+echo "Creating secrets to push images to openshift local registry"
+for image_project in "${image_projects[@]}"
+  do
+    configyaml="\
+    apiVersion: appconnect.ibm.com/v1beta1
+    kind: Configuration
+    metadata:
+      name: ace-policyproject
+      namespace: ${image_project}
+    spec:
+      contents: "$temp"
+      type: policyproject
+    "
+    echo "${configyaml}" > ${PWD}/tmp/policy-project-config.yaml
+    echo "INFO: Output -> policy-project-config.yaml"
+    cat ${PWD}/tmp/policy-project-config.yaml
+    oc apply -f ${PWD}/tmp/policy-project-config.yaml
+done
+
+echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
 
 echo "Waiting for postgres to be ready"
 oc wait -n postgres --for=condition=available deploymentconfig --timeout=20m postgresql
 
-echo "Creating quotes table in postgres samepledb"
-oc exec -n postgres -it $(oc get pod -n postgres -l name=postgresql -o jsonpath='{.items[].metadata.name}') \
-  -- psql -U admin -d sampledb -c \
-'CREATE TABLE QUOTES (
-  QuoteID SERIAL PRIMARY KEY NOT NULL,
-  Name VARCHAR(100),
-  EMail VARCHAR(100),
-  Address VARCHAR(100),
-  USState VARCHAR(100),
-  LicensePlate VARCHAR(100),
-  ACMECost INTEGER,
-  ACMEDate DATE,
-  BernieCost INTEGER,
-  BernieDate DATE,
-  ChrisCost INTEGER,
-  ChrisDate DATE);'
\ No newline at end of file
+echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
+
+declare -a image_projects=("${dev_namespace}" "${test_namespace}")
+
+for image_project in "${image_projects[@]}"
+do
+  ${PWD}/../../products/bash/configure-postgres.sh -n ${image_project}
+  echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
+done
+
+declare -a image_projects=("${dev_namespace}" "${test_namespace}")
+
+for image_project in "${image_projects[@]}"
+do
+  echo "INFO: Creating secret to pull images from the ER"
+
+  oc create -n ${image_project} secret docker-registry ibm-entitlement-key --docker-server=${ER_REGISTRY} \
+    --docker-username=${ER_USERNAME} --docker-password=${ER_PASSWORD} -o yaml | oc apply -f -
+
+  echo "INFO: Creating operator group and subscription in ${image_project}"
+  ${PWD}/../../products/bash/deploy-og-sub.sh -n ${image_project}
```

Comment:
It looks like these won't exit if the child script fails?

Reply:
There is an exit put in place if the child scripts fail